src/views/settings/settings-view.jsx

```diff
@@ -128,7 +153,43 @@ export default function SettingsView() {
 					(c) => item.id === c.id && (item.label !== c.label || item.description !== c.description)
 				)
 			);
+
+			const isReplySignaturePrefisNew =
+				settingsToUpdate.zimbraPrefForwardReplySignatureId &&
+				!settingsToUpdate.zimbraPrefForwardReplySignatureId.includes('-');
+			let setForwardReplySignatureId = '';
+			if (
+				isReplySignaturePrefisNew &&
+				ItemsAdd.length > 0 &&
```

Comment:
I see that these variables were already there, but can you please change the case to be more coherent? we never capitalize variables